static/app/actionCreators/dashboards.tsx

```diff
@@ -40,18 +40,23 @@ export function createDashboard(
   newDashboard: DashboardDetails,
   duplicate?: boolean
 ): Promise<DashboardDetails> {
-  const {title, widgets} = newDashboard;
+  const {title, widgets, projects, environment, period, start, end} = newDashboard;
 
   const promise: Promise<DashboardDetails> = api.requestPromise(
     `/organizations/${orgId}/dashboards/`,
     {
       method: 'POST',
-      data: {title, widgets: widgets.map(widget => omit(widget, ['tempId'])), duplicate},
-      query: {
-        // TODO: This should be replaced in the future with projects
-        // when we save Dashboard page filters. This is being sent to
-        // bypass validation when creating or updating dashboards
-        project: [ALL_ACCESS_PROJECTS],
```

Comment:
Ah, I thought I could remove this but it looks like it prevents us from creating a dashboard with a project that isn't ours. I need to check if we want to let users create dashboards if they select a project that they aren't a member of.

Reply:
I added this back, but instead of giving `ALL_ACCESS_PROJECTS`, I passed down the projects that have been selected.